### ISSUE TITLE
Keyboard doesn't show when phone is rotated

### ISSUE THREAD

**Issue by jvcalassio** [joaocout/terminho#7]
The keyboard is not visible when the phone is in landscape mode.
The screen is also not scrollable

**Comment by joaocout** [comment]
I think I'll just lock it in portrait mode for now...
When used in landscape mode, the original [Termo](https://term.ooo/) web app gets kinda weird, the Grid gets very small, and the Keyboard very wide.
A way to fix this without messing too much with the Keyboard and Grid sizes, and without making the screen scrollable, would be to render the Keyboard and Grid components side by side (?).

Thanks for testing the app, and for reporting this!

**Comment by jvcalassio** [comment]
Yeah, rendering the components side by side seems like a great way to handle the issue!

It's fine if the app is locked on portrait, tho. It doesn't really need to rotate and work on landscape (I haven't tested it, but the current layout seems to work on tablets too)
